addons/l10n_ec/data/account_data.xml

```diff
@@ -1,28 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<odoo>
-    <data noupdate="1">
-
-        <!-- Account Tax Group -->
-        <record id="tax_group_ret_8" model="account.tax.group">
```

Comment:
can we keep the xml_ids of the tax groups as much as possible the same as before?

Reply:
Issues (possible) when migrating from V14 to V15.

Reply:
reolved